src/atoms/TextArea/TextArea.example.mdx

```diff
@@ -0,0 +1,5 @@
+import TextArea from '@atoms/TextArea';
+
+# TextArea Test
+
```

Comment:
Because more than us 3 (@Brookeruu, @CSKingMartin, and you) will be using this could you provide the example with:
- information around the max-character count
- "multi-line plain-text editing control"

Reply:
Also - it would be nice if the max character count could be passed as props, and if you could detail that here in the example that would be ✨